src/couch_epi/src/couch_epi_util.erl

```diff
@@ -22,7 +22,7 @@ module_version(Module) ->
     VSNs.
 
 hash(Term) ->
-    <<SigInt:128/integer>> = couch_hash:md5_hash(term_to_binary(Term)),
+    <<SigInt:128/integer>> = couch_hash:md5_hash(term_to_binary(Term, [{minor_version, 1}])),
```

Comment:
Why not use `?term_to_bin()` here?
```erl
<<SigInt:128/integer>> = couch_hash:md5_hash(?term_to_bin(Term)),
```

Reply:
I mentioned in the [individual commit](https://github.com/apache/couchdb/pull/4581/commits/14d8f8f1d1fb67576561f52106a85f929213fa03) that it's not possible to import `couch_db.hrl` into that file.

Reply:
Maybe add the same macro into `couch_epi.hrl`?

Reply:
Done in https://github.com/apache/couchdb/pull/4581/commits/e5a9bdd4d018695cfddece67e551f97f683fd0b5